app/services/send_chase_email.rb

```diff
@@ -0,0 +1,6 @@
+class SendChaseEmail
+  def perform(reference:)
+    RefereeMailer.reference_request_chaser_email(reference.application_form, reference)
+    ChaserSent.create(chased: reference, chaser_type: :referee_mailer_reference_request_chaser_email)
```

Comment:
```suggestion
    ChaserSent.create!(chased: reference, chaser_type: :referee_mailer_reference_request_chaser_email)
```

On second look perhaps `referee_mailer_reference_request_chaser_email` is a little wordy? Perhaps just `reference_request`, since it's clear it's an email and chaser.